RiotSwiftUI/Modules/Common/Util/MultilineTextField.swift

```diff
@@ -76,7 +76,7 @@ struct MultilineTextField: View {
             .overlay(rect.stroke(borderColor, lineWidth: borderWidth))
             .introspectTextView { textView in
                 textView.textColor = UIColor(textColor)
-                textView.font = theme.fonts.uiFonts.callout
+                textView.font = .element.callout
```

Comment:
It is no longer necessary to have a `uiFonts` property on the SwiftUI themes as these can be referenced statically now.